frequency-chain/style-guide
# 7 footer

# Problem

Need a footer and storybook for footer.

# Solution

- created footer component and storybook
- created dependencies (icon button, etc)
- Added new icons
- text and logo gets smaller in mobile

## Steps to Verify:

1. Run locally and try in different screen sizes
2. check links

## Screenshots (optional):
<img width="894" alt="Screenshot 2024-10-28 at 10 31 54 AM" src="https://github.com/user-attachments/assets/a6f0f473-c6bb-4030-a2df-9e164b450d67">
